assets/app/scripts/extensions/javaLink.js

```diff
@@ -19,10 +19,10 @@ angular.module('openshiftConsoleExtensions', ['openshiftConsole'])
                   .segment('proxy')
                   .segment('namespaces').segment(namespace)
                   .segment('pods').segment(podId);
-      return kubeProxyURI.toString();
+      return kubeProxyURI;
     };
   })
-  .run(function(ProxyPod, BaseHref, HawtioExtension, $templateCache, $compile) {
+  .run(function(ProxyPod, BaseHref, HawtioExtension, $templateCache, $compile, LocalStorageUserStore) {
```

Comment:
can't assume this is the user store being used... can call `$injector.get(AuthService.UserStore())` to get the configured user store